assets/src/infrastructure/ui/inputs/dateTime/Timepicker/index.tsx

```diff
@@ -1,32 +1,46 @@
 import React, { useCallback } from 'react';
 import classNames from 'classnames';
-import { TimePicker as BaseUITimePicker } from 'baseui/timepicker';
+// @ts-ignore
+import { default as ReactTimePicker } from 'react-time-picker';
 
-import { TimepickerProps } from '../types';
-import withBaseProvider from '../withBaseProvider';
+import { CloseOutlined, ClockOutlined } from '@appDisplay/icons/svgs';
+import { TimePickerProps } from '../types';
 
 import './style.scss';
 
-const Timepicker: React.FC<TimepickerProps> = ({ onChange, onChangeValue, ...props }) => {
+const Timepicker: React.FC<TimePickerProps> = ({ onChange, onChangeValue, ...props }) => {
+	const newDate = props?.value;
 	const className = classNames('ee-input-base-wrapper ee-time-picker', props.className);
-	const onChangeHandler: TimepickerProps['onChange'] = useCallback(
-		(date) => {
+	const onChangeHandler: TimePickerProps['onChange'] = useCallback(
+		(time) => {
+			// incoming value from timepicker is 24hr time like "17:00"
+			const timeParts: string[] = time.split(':');
+			const hours = parseInt(timeParts[0]);
+			const minutes = parseInt(timeParts[1]);
```

Comment:
It may not be good to use `.split()`, it's better to use `parse` from `date-fns` and pass the given format from `constants` package.

Reply:
Here you go
![image](https://user-images.githubusercontent.com/18226415/87025578-3b599c00-c1f8-11ea-8672-2aa1934964d9.png)

Reply:
Click on that close icon on TimePicker to see that error.

Reply:
good catch thnx 👍